pages/common/most.md

```diff
@@ -0,0 +1,35 @@
+# most
+
+> Open one or several files for interactive reading, allowing scrolling and search.
+
+- Open a file:
+
+`most {{source_file}}`
+
+- Open several files:
+
+`most {{file1}} {{file2}}`
```

Comment:
Generally, our convention is `{{path/to/file1}}` etc. Could you update the page to match please?